src/ray/core_worker/test/actor_manager_test.cc

```diff
@@ -463,6 +463,63 @@ TEST_F(ActorManagerTest, TestNamedActorIsKilledBeforeSubscribeFinished) {
   ASSERT_TRUE(actor_manager_->GetCachedNamedActorID(cached_actor_name).IsNil());
 }
 
+TEST_F(ActorManagerTest, TestActorDeathCauseBackwardCompatibility) {
```

Comment:
QQ: why are you adding this to core_worker/* tests? I actually wonder if we can just gather all export compatibility tests into a single test file?

Reply:
We can move this test, which folder would you recommend? This test specifically is not only for export API because `ActorDeathCause` should be backward compatible for the state API as well